msg/tools/generate_microRTPS_bridge.py

```diff
@@ -180,6 +185,10 @@ def get_absolute_path(arg_parse_dir):
 
 uorb_templates_dir = os.path.join(msg_folder, args.uorb_templates)
 urtps_templates_dir = os.path.join(msg_folder, args.urtps_templates)
+# parse yaml file into a map of ids
+rtps_ids = px_generate_uorb_topic_helper.parse_yaml_msg_id_file(
```

Comment:
Can you move the definition of `parse_yaml_msg_id_file` to this script, since it's only used here?

Reply:
done